src/averageAges.js

```diff
@@ -20,6 +20,11 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const men = century ? people.filter(person => person.sex === 'm'
+  && Math.ceil(person.died / 100) === century)
+    : people.filter(person => person.sex === 'm');
```

Comment:
[checklist point 2](https://github.com/mate-academy/js_average-ages/blob/master/checklist.md)
```suggestion
  const men = century 
    ? people.filter(person => (
      person.sex === 'm' && Math.ceil(person.died / 100) === century
    ))
    : people.filter(person => person.sex === 'm');
```